ai-service/src/ai_service/db_setup/__init__.py

```
@@ -0,0 +1,17 @@
+"""
+
+from .setup import set_repo_context, get_collection
+from .store_embeddings import add_chunks
+from .query_embeddings import query_chunks
+
+__all__ = [
+    "set_repo_context",
+    "get_collection",
+    "add_chunks",
+    "query_chunks",
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)